web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/UtilsApiController.java

```diff
@@ -3,15 +3,18 @@
 import org.mskcc.cbio.oncokb.apiModels.ActionableGene;
 import org.mskcc.cbio.oncokb.apiModels.AnnotatedVariant;
 import org.mskcc.cbio.oncokb.model.*;
+import org.mskcc.cbio.oncokb.model.oncotree.TumorType;
 import org.mskcc.cbio.oncokb.util.CancerGeneUtils;
 import org.mskcc.cbio.oncokb.util.GeneUtils;
 import org.mskcc.cbio.oncokb.util.MainUtils;
-import org.mskcc.cbio.oncokb.model.oncotree.TumorType;;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 
 import java.util.*;
+
+;
```

Comment:
What's this `;`?

Reply:
Oh, probably the autoformatting moved the top double semicolon down. I will update.